ocl-icd/ocl-icd.spec

```diff
@@ -0,0 +1,55 @@
+Name:                ocl-icd
+Version:             2.2.12
+Release:             2
+Summary:             OpenCL ICD Bindings
+License:             BSD
+URL:                 https://github.com/OCL-dev/ocl-icd
+Source0:             https://github.com/OCL-dev/ocl-icd/archive/v%{version}.tar.gz
```

Comment:
Hm! I've been told that Copr needs the sources tar.gz has to be next to the .spec file but it seems it could also use external urls!

Reply:
Can we use the sources tar.gz which  next to the .spec file？I found in the rpm guide, “This should point to an accessible and reliable storage of the archive, for example, the upstream page and not the packager's local storage.”
![image](https://github.com/bio-arm/rpms/assets/134943615/482edd11-9c46-4ac2-aca4-236f270a6f78)

Reply:
There is an internal restriction. @Yikun explained me today that using external url like here is OK for EUR, but it is not allowed for the official OS packages.
It is OK to use external urls!

Reply:
That means can I use the source from GitHub? I found that some sources are through FTP. May I ask you how to find the FTP address of a package?

Reply:
Usually I go to the project's website and use the download link listed there.
If the project has only a Github repo then I look into the Github `Releases` .

Since we usually copy the RPM files from Fedora, like https://src.fedoraproject.org/rpms/openbabel/tree/rawhide, we can just use whatever someone else already set at `Source0`